queue/queue.go

```diff
@@ -35,11 +38,24 @@ type MetricsSource interface {
 }
 
 func New() (*Queue, error) {
-	return &Queue{}, nil
+	eventQueue := memqueue.NewQueue(logp.L(), memqueue.Settings{
+		Events: 1024,
+		// The event count and timeout for queue flushes is hard-coded to a placeholder
+		// for now, since that's what the existing Beats API understands. The plan is
+		// for these parameters to instead be controlled by the output as it reads
+		// the queue.
```

Comment:
Can this be an issue so we don't forget to do it?

Reply:
https://github.com/elastic/elastic-agent-shipper/issues/58